docs/topics/gradle/get-started-with-jvm-gradle-project.md

```diff
@@ -112,4 +112,5 @@ Learn more about:
 * [Gradle build file properties](https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#N14E9A).
 * [Targeting different platforms and setting library dependencies](gradle-configure-project.md).
 * [Compiler options and how to pass them](gradle-compiler-options.md).
-* [Incremental compilation, caches support, build reports, and the Kotlin daemon](gradle-compilation-and-caches.md).
\ No newline at end of file
+* [Incremental compilation, caches support, build reports, and the Kotlin daemon](gradle-compilation-and-caches.md).
+* [Support for Gradle plugin variants](gradle-plugin-variants.md).
```

Comment:
I am not sure newcomers should learn about Gradle plugin variants. I would propose to remove this link